rest-fights/src/main/resources/application.properties

```diff
@@ -14,15 +14,26 @@ quarkus.http.test-port=0
 ## Mongo
 quarkus.mongodb.metrics.enabled=true
 quarkus.mongodb.database=fights
-
+#quarkus.mongodb.connection-string=mongodb://localhost:52003
+#quarkus.mongodb.credentials.username=superfight
+#quarkus.mongodb.credentials.password=superfight
 ## Liquibase Mongo
 quarkus.liquibase-mongodb.migrate-at-start=true
 
 ## CORS
 quarkus.http.cors=true
-quarkus.http.cors.origins=*
-
+quarkus.http.cors.origins=http://localhost:8082,http://localhost:8080,http://localhost:8086
```

Comment:
This shouldn't be here. The problem is when you deploy to k8s, you don't know ahead of time what the origins are, and therefore this will break.

Reply:
`quarkus.http.cors.access-control-allow-credentials=true` my understanding was when I enable this, I would need to add the `quarkus.http.cors.origins` as seen in some of the examples here:https://quarkus.io/guides/security-cors#quarkus-vertx-http-config-group-cors-cors-config_quarkus-http-cors-access-control-allow-credentials. I guess i'm not sure the best way to handle this now or if my understanding was incorrect of whether or not I need `quarkus.http.cors.access-control-allow-credentials=true` when the frontend has `credentials:include` (which is needed for the WebAuthn to work)

Reply:
I understand its purpose, but unfortunately doing this breaks one of the core tenants of the superheroes app - it needs to be "stupid simple" to run and deploy.

By having this requirement means that the fully-qualified URLs for all the apps have to be known ahead of deploying the app to a platform like kubernetes. Rather than just `oc apply`'ing a yaml file, a user would first have to hand edit a `ConfigMap` within one of the files in https://github.com/quarkusio/quarkus-super-heroes/tree/main/deploy/k8s and put in all the fully-qualified URLs.

_Maybe_ we could get away with specifying something like `http://auth-server:8086` which uses the kubernetes service name, but we certainly wouldn't know the fully-qualified browser URL of the ui app at deployment time.